lib/squeel/nodes/predicate_methods.rb

```diff
@@ -20,6 +20,14 @@ def eq(value = :__undefined__)
         end
       end
 
+      def nil?
+        eq(nil)
+      end
+
+      def present?
```

Comment:
This doesn't match AS `present?` behavior, which would not match an empty string. I think this would be potentially confusing. However, beyond this I don't think overloading these methods makes sense for the Squeel DSL, anyway.